ansible/roles/test/files/ptftests/vrf_test.py

```diff
@@ -24,7 +24,7 @@
 
 class FwdTest(FibTest):
     _required_params = [
-        'router_mac',
+        'router_macs',
```

Comment:
Why change from mac to macs? Is there more than 1 mac passed in here?

Reply:
There is only 1 mac passed.
Vrf tests inherit FibTest which was updated in [PR](https://github.com/Azure/sonic-mgmt/pull/2740) and now requires such params as: 'router_macs', 'fib_info_files', 'ptf_test_port_map'.  Without passing these params Vrf tests will fail.